bibliopixel/remote/control.py

```diff
@@ -104,13 +110,32 @@ def __init__(self, layout, animations,
         self.handlers = {
             'run_animation': self.run_animation,
             'stop_animation': self.stop_animation,
-            'get_config': self.get_config
+            'get_config': self.get_config,
+            'trigger_animation': self.run_animation
         }
 
+        self.trigger_procs = {}
+        self.triggers = {}
+        for t in triggers:
+            typename = t['typename']
+            importer.import_symbol(typename)  # attempt early to fail early
+            if typename not in self.triggers:
+                self.triggers[typename] = []
+            t.pop('typename')
+            self.triggers[typename].append(t)
+
+        for typename, configs in self.triggers.items():
+            trigger_args = (typename, self.q_recv, configs)
+            self.trigger_procs[typename] = multiprocessing.Process(target=trigger_process.run_trigger,
+                                                                   args=trigger_args)
+
     def cleanup(self, clean_layout=True):
         self.q_recv.close()
-        self.q_send.close()
+        for q in self.resp_qs.values():
+            q.close()
         self.server.terminate()
+        # for _, t in self.trigger_procs.items():
```

Comment:
Oh, shit... that was supposed to be actually used. I commented it for testing something. It needs to go back in.